recipes/blend2d/all/conanfile.py

```diff
@@ -67,15 +74,17 @@ def generate(self):
         tc.variables["BLEND2D_STATIC"] = not self.options.shared
         tc.variables["BLEND2D_NO_STDCXX"] = False
         tc.variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
+        tc.variables["BLEND2D_EXTERNAL_ASMJIT"] = True
```

Comment:
Was the library using a vendorized version all this time for old releases? Ouch!

Reply:
Nope, we apply patch for older versions to avoid the vendorized asmjit.